qemu/tests/fio_perf.py

```diff
@@ -0,0 +1,259 @@
+import os
+import re
+import time
+import threading
+import logging
+
+from autotest.client import utils
+
+from avocado.utils import process
+from avocado.core import exceptions
+
+from virttest import utils_misc, utils_test
+
+
+def format_result(result, base="12", fbase="2"):
+    """
+    Format the result to a fixed length string.
+
+    :param result: result need to convert
+    :param base: the length of converted string
+    :param fbase: the decimal digit for float
+    """
+    if isinstance(result, str):
+        value = "%" + base + "s"
+    elif isinstance(result, int):
+        value = "%" + base + "d"
+    elif isinstance(result, float):
+        value = "%" + base + "." + fbase + "f"
+    return value % result
+
+
+def check_status(session, timeout, num):
+    """
+    Output disk info including disk status
+
+    :param session: VM session
+    :param timeout: Timeout in seconds
+    :param num: The number of data disks
+    """
+
+    disk_status_cmd = "echo list disk > cmd && echo exit >>"
+    disk_status_cmd += " cmd && diskpart /s cmd"
+    disks = []
+    end_time = time.time() + timeout
+    while time.time() < end_time:
+        disks_str = session.cmd_output_safe(disk_status_cmd)
+        logging.info("disks_str is %s" % disks_str)
+        disks = re.findall("Disk %s.*\n" % num, disks_str)
+        if not disks:
+            continue
```

Comment:
I think no need, normally, when run cmd here, disks have been ready on windows, so disks variable is not null. Even if there is an exception, we can not sure how long is the best time to sleep, so it is ok to get disks value via while.